src/products/firefox_android/how_to_localize.md

```diff
@@ -2,16 +2,16 @@
 
 Firefox for Android is on a four week rapid release cycle. This means that usually there is a new release going out every four weeks.
 
-Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that localizers can complete their work. The deadline in Pontoon is an indicator of when the next localization cut-off date is for the upcoming release - in other words, the last day to get localization updates into that release.
+Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that localizers can complete their work. The deadline in [Pontoon](https://pontoon.mozilla.org/projects/firefox-for-android/) is an indicator of when the next localization cut-off date is for the upcoming release - in other words, the last day to get localization updates into that release.
```

Comment:
Is the 2+2 model still a thing, given they follow the release cycle of Firefox? I would expect nightly to be changing for 4 weeks, and then have a period without changes in Beta?

Reply:
Oops, you are right! thanks for catching that. I'll update it